src/components/forms/SubnetForm.js

```diff
@@ -39,6 +39,9 @@ class SubnetForm extends React.Component {
   }
 
   render() {
+    function cidrIsValid(cidr) {
```

Comment:
do we want this function inside the render? we could make it like the handleToggle function